src/common/file_entry.rs

```diff
@@ -4,6 +4,8 @@ use std::path::PathBuf;
 
 use crate::common::*;
 
+const EXCLUDED_FILES: &[&str] = &[".envrc"];
```

Comment:
```suggestion
const IGNORE_LIST: &[&str] = &[".envrc"]; // https://direnv.net
```
I like the name `IGNORE_LIST` better. What do you think about it?
Also, it would be great if you added a comment why we ignore this file name.

Reply:
I used an existing term "exclude" because I see this new feature as a variation of it.
User "excludes" files explicitly by passing command line options.
We "exclude" files implicitly.
Am I missing an important distinction between what user does and what we do?
In other words, what warrants adding new term "ignore" for this situation?

Reply:
Okay, let it be 🙂